### PR TITLE
MMB: Add visa required warning

### PR DESCRIPTION
**Pull request by tbergq**

### CODE REVIEW

File: app/MMB/src/scenes/passenger/Passenger.js [hunk 1]
```diff
@@ -48,40 +49,43 @@ export class Passenger extends React.Component<Props> {
...
-            value={<Translation passThrough={idNumber} />}
-          />
-          <Column
-            title={<Translation id="mmb.passenger.insurance" />}
-            value={<Translation passThrough={insuranceType} />}
-          />
-        </Row>
-      </SimpleCard>
+      <React.Fragment>
+        <SimpleCard style={styles.card}>
```
Comment (mrtnzlml) [inline app/MMB/src/scenes/passenger/Passenger.js]: Did you try it on Android? `SimpleCard` component is with radiuses so I cannot imagine this works well together... 🙂

File: ios/reactNativeApp.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -480,46 +480,6 @@
 			shellPath = /bin/sh;
 			shellScript = "export NODE_BINARY=node\n../node_modules/react-native/scripts/react-native-xcode.sh";
 		};
-		2711985D20B1EA8100C6292B /* Bundle React Native dependencies */ = {
```
Comment (mrtnzlml) [inline ios/reactNativeApp.xcodeproj/project.pbxproj]: Is this intentional?

Comment (tbergq) [reply ios/reactNativeApp.xcodeproj/project.pbxproj]: 🤷‍♂️ happens when I build

Comment (tbergq) [reply ios/reactNativeApp.xcodeproj/project.pbxproj]: This is ok according to @ferrannp

**Comment by tbergq** [comment]
@mrtnzlml please check again

File: app/MMB/src/components/Alert.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+      return null;
+    case 'success':
+      // TODO
+      return null;
+    default:
+      return null;
+  }
+}
+
+const RED = '#d0021b';
```
Comment (mrtnzlml) [inline app/MMB/src/components/Alert.js]: Should we use new brand colors here or not? https://zpl.io/2GmG6mW

screen shot 2018-05-23 at 09 52 43

Comment (tbergq) [reply app/MMB/src/components/Alert.js]: 🤷‍♂️ are there any designs for these `alerts` in the new design?

Comment (mrtnzlml) [reply app/MMB/src/components/Alert.js]: Only this: https://zpl.io/bzYGpjz

The colors are different though...

Comment (tbergq) [reply app/MMB/src/components/Alert.js]: The color codes in the new `alerts` are the same as before

File: app/MMB/src/components/Alert.js [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+      <Icon name="warning" size={18} color={RED} />
+    </View>
+    <View>
+      <Text style={[styles.titleText, styles.titleDanger]}>{props.title}</Text>
+      <View style={styles.body}>{props.children}</View>
+    </View>
+  </View>
+);
+
+export default function Alert(props: Props) {
```
Comment (mrtnzlml) [inline app/MMB/src/components/Alert.js]: Is this component necessary? Cannot we just use the "danger alert" directly?

Comment (tbergq) [reply app/MMB/src/components/Alert.js]: The idea was to expose one component, with 3 types, then that component selects the necessary one. 

Basically these components will be the same, with different styling.

**Pull request merged**